builder/bundles.go

```diff
@@ -447,6 +447,22 @@ func installBundleToFull(packagerCmd []string, buildVersionDir string, bundle *b
 		for p := range bundle.AllPackages {
 			args = append(args, p)
 		}
+
+		// Retry RPM downloads to avoid timeout failures due to slow network
+		retries := 0
+		downloadArgs := merge(args, "--downloadonly")
+
+	DOWNLOAD_RPMS:
+		_, err = helpers.RunCommandOutputEnv(downloadArgs[0], args[1:], []string{"LC_ALL=en_US.UTF-8"})
+		if err != nil {
+			if retries < 2 {
+				retries++
+				fmt.Println("Retrying after failed RPM download")
+				goto DOWNLOAD_RPMS
```

Comment:
I don't like gotos with the only exception for going to the end of a function on error handling.
It's a lot easier to read if you have

    while (retry < max_retries) {
         ...
         if (err == nil) break; //Or return if you create another function
    }

Reply:
I was going back-and-forth on this. I can switch it to a loop implementation.

Reply:
I'm with otavio here, specially considering that feels out of place compared to the rest of the code base.